sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/optimizer/EliminateSortsSuite.scala

```diff
@@ -97,12 +97,34 @@ class EliminateSortsSuite extends PlanTest {
     comparePlans(optimized, correctAnswer)
   }
 
-  test("remove redundant order by") {
```

Comment:
We need to modify the existing test `test("remove redundant order by") {`? If the behaviour of the test does not change, I think it would be better to avoid updating it where possible.

Reply:
The test was actually changed from `orderBy` (global) to `sortBy` (local) so the test title was updated to "remove redundant sort by"